#Created by Quincy Fleming.
#Disables debug info without using flag NDEBUG as unless forced it will be ignored in favor of the user's specified compiler flags 
--- a/src/gui/include/all/clipboard/logging.hpp
+++ b/src/gui/include/all/clipboard/logging.hpp
@@ -17,7 +17,6 @@
 #include <iostream>
 #include <streambuf>
 
-#if defined(NDEBUG)
 class NullBuffer : public std::streambuf {
 public:
     int overflow(int c) override { return c; }
@@ -26,6 +25,3 @@
 static NullBuffer nullBuffer;
 static std::ostream nullStream(&nullBuffer);
 static std::ostream& debugStream = nullStream;
-#else
-static std::ostream& debugStream = std::cerr;
-#endif
